src/test/java/com/rick/racing/model/CarNameTest.java

```diff
@@ -0,0 +1,20 @@
+package com.rick.racing.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class CarNameTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"value", "test1", "hello", "sdf"})
+    @DisplayName("인스턴스 정상 생성 확인")
+    public void create(String name) {
+        final CarName carNameA = CarName.create(name);
+        final CarName carNameB = CarName.create(name);
+
+        assertThat(carNameA.equals(carNameB)).isTrue();
```

Comment:
다음과 같이 줄여서 표현해볼 수도 있을 것 같아요.
```suggestion
        assertThat(carNameA).isEqualTo(carNameB);
```